prep-css-display/display.css

```diff
@@ -0,0 +1,24 @@
+div {
+  border: 4px solid blue;
+}
+
+span {
+  border: 4px solid green;
+  height: 100px;
+  width: 100px;
+}
+
+button {
+  border: 4px solid red;
+  height 40px
+  width: 200px;40pxwidth40pxwidth40pxwidth40pxwidth40pxwidth
+}
```

Comment:
Not too sure what happened here 😅 be sure to include `:` a colon and `;` a semicolon on your line 13